Source/towners.cpp

```diff
@@ -1176,47 +1178,44 @@ void TalkToTowner(int p, int t)
 				SpawnUnique(UITEM_BOVINE, towner[t]._tx + 1, towner[t]._ty);
 				RemoveInvItem(p, i);
 				qt = TEXT_JERSEY8;
-				quests[Q_JERSEY]._qactive = 3;
+				quests[Q_JERSEY]._qactive = QUEST_DONE;
 			} else if (PlrHasItem(p, IDI_RUNEBOMB, &i)) {
 				qt = TEXT_JERSEY5;
-				quests[Q_JERSEY]._qactive = 2;
+				quests[Q_JERSEY]._qactive = QUEST_ACTIVE;
 				quests[Q_JERSEY]._qvar1 = 1;
 				quests[Q_JERSEY]._qmsg = TEXT_JERSEY4;
 				quests[Q_JERSEY]._qlog = 1;
 			} else {
 				switch (quests[Q_JERSEY]._qactive) {
-				case 0:
-					qt = TEXT_JERSEY1;
-					quests[Q_JERSEY]._qactive = 7;
-					break;
-				case 1:
+				case QUEST_NOTAVAIL:
+				case QUEST_INIT:
 					qt = TEXT_JERSEY1;
-					quests[23]._qactive = 7;
+					quests[Q_JERSEY]._qactive = QUEST_HIVE_TEASE1;
 					break;
-				case 2:
+				case QUEST_ACTIVE:
 					qt = TEXT_JERSEY5;
 					break;
-				case 3:
+				case QUEST_DONE:
 					qt = TEXT_JERSEY1;
 					break;
-				case 7:
+				case QUEST_HIVE_TEASE1:
 					qt = TEXT_JERSEY2;
-					quests[Q_JERSEY]._qactive = 8;
+					quests[Q_JERSEY]._qactive = QUEST_HIVE_TEASE2;
 					break;
-				case 8:
+				case QUEST_HIVE_TEASE2:
 					qt = TEXT_JERSEY3;
-					quests[Q_JERSEY]._qactive = 9;
+					quests[Q_JERSEY]._qactive = QUEST_HIVE_ACTIVE;
 					break;
-				case 9:
+				case QUEST_HIVE_ACTIVE:

```

Comment:
Amazing boost in readability on this function. Great stuff.

Reply:
Much more to some in this function :)